administrator/components/com_content/models/forms/article.xml

```diff
@@ -691,6 +692,7 @@
 	<fields name="menulink" label="COM_CONTENT_MENU_LINK_OPTIONS" addfieldpath="/administrator/components/com_menus/models/fields">
 		<field
 				name="menutitle"
+				id="menutitle"
```

Comment:
It is suitable. but you just don't need id. not needed anything additional code when you can get using `name` and it is also unique.